### PR TITLE
Feature/fix camera issue android4

### PR DESCRIPTION
**Pull request by wpbs**
#9 fixed

### CODE REVIEW

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java [hunk 1]
```diff
@@ -185,13 +187,22 @@ private void dispatchTakePictureIntent() {
...
+                    Log.i("DocumentActivity", "File stored");
+                    takePictureIntent.putExtra(MediaStore.EXTRA_OUTPUT, photoURI);
+                }
                 Log.i("DocumentActivity", "Starting take picture intent");
                 startActivityForResult(takePictureIntent, REQUEST_TAKE_PHOTO);
                 Log.i("DocumentActivity", "Done with take picture intent");
             }
+        } else {
+            Log.i("DocumentActivity", "No camera app available?!");
+            // TODO: present toast?
```
Comment (dchri) [inline app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java]: is this Todo also on the issues/bug board?

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java [hunk 2]
```diff
@@ -235,12 +246,47 @@ private void resizePictureToThumbnail(final String originalFileLocation) {
...
+                    Log.i("DocumentActivity", "Writing to output stream");
+                    FileOutputStream out = new FileOutputStream(mCurrentPhotoPath);
+
+                    Log.i("DocumentActivity", "Compressing output");
+                    bmp.compress(Bitmap.CompressFormat.PNG, 100, out);
+                    out.flush();
+                    out.close();
+                    Log.i("DocumentActivity", "Done writing to stream");
+                } catch (Exception e) {
+                    Log.e("DocumentActivity", "Unable to save image: " + e.getLocalizedMessage());
```
Comment (dchri) [inline app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java]: Is this enough, or do you need the stacktrace for debugging purposes?

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java (hunk 1 above)
Comment (wpbs) [reply app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java]: Created #16 for this

File: app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java (hunk 2 above)
Comment (wpbs) [reply app/src/main/java/nl/devolksbank/nlhelpu/activity/DocumentActivity.java]: done

**Pull request merged**